cross-modal-search/pods/executors.py

```diff
@@ -8,7 +8,17 @@
 from PIL import Image
 import io
 from jina import Executor, DocumentArray, requests, Document
-from jina.helloworld.chatbot.my_executors import _norm, _ext_B, _ext_A, _cosine
+from jina.helloworld.chatbot.executors import _norm, _ext_B, _ext_A, _cosine
+
+
+class ReciprocalRankEvaluator(Executor):
+    def rank_evaluate(self, actual: Sequence[Union[str, int]], desired: Sequence[Union[str, int]], **kwargs):
```

Comment:
It needs the `request` decorator

Reply:
and please review `Executor` cookbook, this will never work with 2.0

Reply:
there is request routing in yaml

Reply:
the actual ranking and evaluating is still WIP